cli/auth_nkey_command.go

```diff
@@ -256,3 +272,106 @@ func (c *authNKCommand) verifyAction(_ *fisk.ParseContext) error {
 
 	return nil
 }
+func (c *authNKCommand) sealAction(_ *fisk.ParseContext) error {
+	var err error
+	var kp nkeys.KeyPair
+
+	keyData, err := c.readKeyFile(c.keyFile)
+	if err != nil {
+		return err
+	}
+
+	// try it as public, then as seed
+	kp, err = nkeys.FromPublicKey(string(keyData))
+	if errors.Is(err, nkeys.ErrInvalidPublicKey) {
+		kp, err = nkeys.FromSeed(keyData)
+	}
+	if err != nil {
+		return err
+	}
+
+	content, err := os.ReadFile(c.dataFile)
+	if err != nil {
+		return err
+	}
+
+	if nkeys.IsValidPublicCurveKey(c.counterpartKey) {
+		encryptedData, err := kp.Seal(content, c.counterpartKey)
+		if err != nil {
+			return err
+		}
+		if c.useB64 {
+			encryptedData = []byte(base64.StdEncoding.EncodeToString(encryptedData))
+		}
+
+		if c.outFile != "" {
```

Comment:
like above I'd `if c.outFile == "" { print/return }` and then have the bigger code block unindented and no need for an else then